src/basic/env-file.c

```diff
@@ -637,3 +637,12 @@ int write_env_file(int dir_fd, const char *fname, char **headers, char **l) {
         (void) unlinkat(dir_fd, p, 0);
         return r;
 }
+
+int write_vconsole_conf(int dir_fd, const char *fname, char **l) {
+        char **headers = STRV_MAKE(
+                "# Written by systemd-localed(8) or systemd-firstboot(1), read by systemd-localed",
+                "# and systemd-vconsole-setup. It is probably wise not to edit this file manually.",
```

Comment:
i think it's totally Ok to edit this file manually. I'd remove that part.

Reply:
PID 1 reads this too btw.

Reply:
> i think it's totally Ok to edit this file manually. I'd remove that part.

When the file is updated by localed, systemd-vconsole-setup.service is restarted. Should we mention that the service should be also restarted manually when the file is manually edited?

Reply:
> PID 1 reads this too btw.

Does it? I cannot find relevant code. Maybe you confused with locale.conf ?

Reply:
oops, indeed. sorry.